### ISSUE TITLE
something wrong with npm install

### ISSUE THREAD

**Issue by wangling12** [butterproject/butter-desktop#697]
Just npm install 
and
![2018-08-03 19-21-59](https://user-images.githubusercontent.com/24770600/43640765-ba80d664-9753-11e8-9248-5117063b977b.png)

It's going to stay for a long time and download something more than 1GB.
### Technical details
 ubuntu 18.04
4.15.0-29-generic
zsh
npm:6.2.0
node:v9.4.0

**Comment by xaiki** [comment]
please check new devel blanch and close if unrelevant

**Comment by Gabrielvon** [comment]
I got same warnings like what @wangling12 posted though I don't know how large they are. Any solutions for this?

**Comment by btzr-io** [comment]
Here is the link: https://github.com/butterproject/butter-desktop/tree/devel

**Comment by Gabrielvon** [comment]
@xaiki hi, i am relatively new to git and don't really know what exactly to do. Do you mean `git checkout devel` ? I didn't see that the `devel` branch is also cloned after I did `git clone https://github.com/butterproject/butter-desktop.git`.

**Comment by xaiki** [comment]
remove your current tree and make a clean clone.
the `devel` branch is the new default branch so @btzr-io's link is correct

**Comment by Gabrielvon** [comment]
@xaiki finally. I checked the `devel` branch and thanks @btzr-io . However, I still cannot install it correctly. I tried two method as follows and both were failed.

**First method**:
```
yarn
yarn install
```
It returns the errors after I enter `yarn install`. 
> ERROR: install is not COMMAND nor fully qualified CLASSNAME.>


<br><br>
**Second method** which is exactly like #625 and produced similar error. 
```
1. rm -rf ~/.npm/
2. git clone https://github.com/butterproject/butter-desktop.git && cd butter-desktop
3. npm install -g gulp-cli
4. npm install
5. gulp build
6. gulp run
```

At the step 5, I got `No gulpfile found` and it asked me to do `npm install gulp` which I did. And it showed
>found 5 vulnerabilities (1 low, 4 high)
  run `npm audit fix` to fix them, or `npm audit` for details

And I did `npm audit fix --force` and still got `No gulpfile dound`
>gulp -v
[21:13:00] CLI version 2.0.1
[21:13:00] Local version 4.0.0
>npm -v
6.3.0
